ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
The version can't come from the parent because there is no version defined in [jvnet-parent](https://repo1.maven.org/maven2/net/java/jvnet-parent/5/jvnet-parent-5.pom). This means that it's unpredictable which version will be used for the build so the warning makes absolutely sense.